plugins/safebooru.js

```diff
@@ -19,10 +19,10 @@ hoverZoomPlugins.push({
 
         hoverZoom.urlReplace(res,
             'img[src]',
-            /(.*)\/(?:thumbnails|samples)\/(.*)\/(?:thumbnail|sample)_(.*)\.jpg(.*)/,
-            '$1/images/$2/$3.png'
+            /(.*)\/(?:thumbnails|samples)\/(.*)\/(?:thumbnail|sample)_(.*)/,
+            '$1/images/$2/$3'
         );
 
-        callback($(res));
+        callback($(res), this.name);
```

Comment:
I'm just curious, what is this doing? The callback function `imgLinksPrepared` only takes one parameter.

Reply:
It's for debugging, so you know which plugin found the high resolution url (it is not always obvious).
I use a slightly different version of HZ+where this info is handled.

Reply:
I see, thank you.